MachineLearning_Engine/Compute/Structured/Coefficients.cs

```diff
@@ -29,18 +29,18 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 
-namespace BH.Engine.MachineLearning
+namespace BH.Engine.MachineLearning.Structured
```

Comment:
This will need versioning with a `Versioning_41.json` file.

Reply:
Would a PreviousVersion work? Or it should be a versioning json for type?

Reply:
Alas not as you've changed the namespace for a specific part, the only way a previous version would work is if you use one on all methods that have no changed namespace. Have you changed the namespace for all bits or just a few methods?

Reply:
Both namespaces exists just moved from one to the other. Should be just a previous version.

Reply:
Ok, let's try that, should be ok :smile: